maskrcnn_benchmark/utils/metric_logger.py

```diff
@@ -61,3 +65,41 @@ def __str__(self):
                 "{}: {:.4f} ({:.4f})".format(name, meter.median, meter.global_avg)
             )
         return self.delimiter.join(loss_str)
+
+
+class TensorboardLogger(MetricLogger):
+    def __init__(self,
+                 log_dir='logs/maskrcnn-benchmark',
```

Comment:
can we avoid having a default argument for this value? I think it's better to keep a default argument in the place where it's called

Reply:
sure, on it